<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExternalProgramsDialog</class>
 <widget class="QDialog" name="ExternalProgramsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>144</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>144</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>External Programs</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QPushButton" name="inkscape_pushButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="inkscape_label">
       <property name="text">
        <string>Inkscape:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="pdf_latex_lineEdit"/>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pdf_latex_pushButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pdf_viewer_pushButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="pdf_viewer_label">
       <property name="text">
        <string>Pdf viewer:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="pdf_viewer_lineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="pdflatex_label">
       <property name="text">
        <string>pdflatex:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="inkscape_lineEdit"/>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="python_pushButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="python_lineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="python_label">
       <property name="text">
        <string>python 3:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="defaultButton">
       <property name="text">
        <string>Default</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>inkscape_lineEdit</tabstop>
  <tabstop>inkscape_pushButton</tabstop>
  <tabstop>pdf_latex_lineEdit</tabstop>
  <tabstop>pdf_latex_pushButton</tabstop>
  <tabstop>pdf_viewer_lineEdit</tabstop>
  <tabstop>pdf_viewer_pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExternalProgramsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExternalProgramsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
